authentication/graphql.py

```diff
@@ -81,6 +84,36 @@ def resolve_balance(data, _args, _context, _info):
         balance = client.Accounts.get(data.access_token)['accounts'][0]['balances']['available']
```

Comment:
In my balances method I get all balances (rather than just first); think we should remain consistent, and that it would be good to cover all balances. We could do that two ways: show multiple, or sum them all. I think the sum is a better idea for this. Should just be able to use the same code as I use with a final modification: 
`          response = client.Accounts.balance.get(user_bank.access_token)
            sum = 0
            for account in response['accounts']:
                if account['subtype'] == 'cd':
                    sum += account['balances']['current']
                elif account['subtype'] == 'credit card':
                    sum -= account['balances']['current']
                else:
                    sum += account['balances']['available']
            return sum
`

Reply:
Apparently github doesn't format code :(